.travis.yml

```diff
@@ -46,6 +46,13 @@ jobs:
     - name: storybook-smoke-test (node@production)
       <<: *node-prod
       script: yarn storybook-smoke-test
+    - name: yarn serve (node@production)
+      <<: *node-prod
+      script:
+        # `yarn build` takes some time so let's simulate it instead.
+        - mkdir build && touch build/index.html
```

Comment:
This seems fragile. Even though it might be slower I think this should probably run a real build. This job will run in parallel to the others so it ultimately shouldn't slow us down, I'd think.

Reply:
Ok, that sounds good to me.